### DISCUSSION TITLE
How to create own blocklists?

### DISCUSSION THREAD

**Discussion by Bluscream** [AdGoBye/AdGoBye-Blocklists#6] [Q&A] [answered]
I think i understand part of the syntax now, but what i don't quite understand is how one is getting the names of gameobjects in the first place.
I know it was easy back when we had mods (We could've had a Adblock mod with runtime patching back then, good times).
But i don't quite see how to do it now?
Is one supposed to rip/disassemble a world they want to check for ads, or is there some kind of ingame "object inspector" that will just tell you the name of the GO you're looking at just like the F3 menu in minecraft for example.

**Comment by DubyaDude** [comment, marked as answer]
There is no in-game menu that tells you the game object names.
So, you'll need to find game-object paths through other means.
